README.md

```diff
@@ -1,49 +1,90 @@
-# VMware Tanzu Developer Center
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [About The Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Open Projects, Issues, and Content Backlog](#open-projects-issues-and-content-backlog)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
-## Building the Site
 
-The VMware Tanzu Developer Center uses [Hugo](https://gohugo.io/) to build the site from Markdown files. You'll need to [get Hugo](https://gohugo.io/getting-started/installing/) if you want to build and run the site locally. Make sure you install the extended version with SCSS/SASS support built in.
+## About The Project
 
-### Run locally
+![Tanzu Developer Center Screen Shot](static/images/github/homepage-screenshot.png)
 
-To install the latest version of `hugo` you can use `brew install hugo` if you are on a Mac. This site pins hugo to a specific version (currently 0.82.0) to build so if you're using a different version, your experience may vary.
+The VMware Tanzu Develoer Center is VMware's site built specifically with devlopment teams in mind. Above all, our guiding pribiples for content included on the Tanzu Developer Center is that if the reader is excited about a topic, they don't have to contact a sales rep or purchase a pricey license because all content in the Tanzu Developer Center is immediately actionable, either because it is open source, or there is an easily accessible trial.Above all, we want the Tanzu Developer Center to be a great resource for software development teams. 
```

Comment:
```suggestion
The VMware Tanzu Develoer Center is VMware's site built specifically with development teams in mind. Above all, our guiding principle for content included on the Tanzu Developer Center is that if the reader is excited about a topic, they don't have to contact a sales rep or purchase a pricey license because all content in the Tanzu Developer Center is immediately actionable, either because it is open source, or there is an easily accessible trial. Above all, we want the Tanzu Developer Center to be a great resource for software development teams. 
```